### ISSUE TITLE
Dynamic Loading Of Levels

### ISSUE THREAD

**Issue by Ambamore2000** [Ambamore2000/groupproject-team-1-b-aj#1]
Currently, the levels are all stored in one class (The MapGraphics.java class) which can theoretically become an issue during runtime when loading a significant amount of levels at once. Rather than creating them at runtime, I propose to load them dynamically by:

Creating a new LevelLoader.java class that will act as a way to read in a file (JSON? Custom Parser?) for when a level is to be loaded.
In order to do such, the mousePressed() function in the MapGraphics.java class will have to be altered to utilize the LevelLoader.java class  for loading levels dynamically.

**Comment by Ambamore2000** [comment]
Same issue with Enemies, could potentially be too large of a task to fix up the loading of both the Enemies and Levels; Something to think about though.

**Comment by Ambamore2000** [comment]
Same issue with Rewards, realized:
I will have to combine the dynamic aspect for all 3 parts: Levels, Enemies, And Rewards.

**Comment by Ambamore2000** [comment]
Ideally this would be my go-to issue to solve for COMP 55 Resurrection Part 2.
Difficult enough for my skill level and will take time to produce proper output in a timely manner.

**Comment by onzfonz** [comment]
Yes, this looks like a great issue!  Go with this one!  Just make sure you do some JSON stuff in there!
